ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/BasicCallbackHandler.java

```diff
@@ -68,7 +64,8 @@ public void handle(final Callback[] callbacks)
             throw new IOException("Unsupported message type: " + toc.getMessageType());
         }
       } else {
-        throw new UnsupportedCallbackException(callback);
+        // We ignore unknown callback types - e.g. Jetty implementation might pass us Jetty specific
```

Comment:
Can we add a log message if this happens? At least a warning, maybe even an error.

Reply:
Let's chat later